src/main/java/com/exactpro/th2/fix/client/Main.java

```diff
@@ -157,32 +160,27 @@ public static void run(Settings settings, MessageRouter<MessageGroupBatch> messa
                            GrpcRouter grpcRouter, Deque<Resources> resources) throws CreatingConfigFileException, ConfigError, IncorrectDataFormat {
 
         File configFile = FixBeanUtil.createConfig(settings);
+        System.out.println(settings);
 
-        StringBuilder stringBuilder = new StringBuilder();
-        Map<SessionID, ConnectionID> connections = new HashMap<>();
-        Map<String, SessionID> sessionIDS = new HashMap<>();
-
-        for (int i = 0; i < settings.sessionsSettings.size(); i++) {
+        Map<SessionID, ConnectionID> connectionIDs = new HashMap<>();
+        Map<String, SessionID> sessionIDs = Settings.getSessionIDs(settings.sessionSettings);
 
-            FixBean fixBean = settings.sessionsSettings.get(i);
-            String sessionAlias = fixBean.getSessionAlias();
-            SessionID sessionID = FixBeanUtil.getSessionID(fixBean);
-            sessionIDS.put(sessionAlias, sessionID);
-            connections.put(sessionID, ConnectionID.newBuilder().setSessionAlias(sessionAlias).build());
-            stringBuilder.append(sessionAlias);
-
-            if (i < settings.sessionsSettings.size() - 1) {
-                stringBuilder.append(":");
-            }
+        for (Map.Entry<String, SessionID> sessionIDEntry : sessionIDs.entrySet()) {
+            String sessionAlias = sessionIDEntry.getKey();
+            SessionID sessionID = sessionIDEntry.getValue();
+            connectionIDs.put(sessionID, ConnectionID.newBuilder().setSessionAlias(sessionAlias).build());
         }
 
         Event curEvent = MessageRouterUtils.storeEvent(eventRouter, Event.start(), null);
-        curEvent.name("FIX client " + stringBuilder + " " + Instant.now());
+        curEvent.name("FIX client " + sessionIDs.keySet().stream()
+                .reduce((resultKey, nextKey) -> resultKey + nextKey)
```

Comment:
> reduce((resultKey, nextKey) -> resultKey + nextKey)

This would result in concatenation of session aliases without any delimiters. Use plain `String.join` instead